### ISSUE TITLE
plugins for postmortem stacks

### ISSUE THREAD

**Issue by davidmarkclements** [davidmarkclements/cute-stack#5]
Any value to having a secondary purpose - persisted stack logs for post mortem analysis

So plugins that actually append to log files, or logger databases etc - the JSON stack kind of alludes to this use case

@bahmutov can you weigh in

**Comment by bahmutov** [comment]
yup, definitely there is value to that. But I would start with a few simpler things. Like showing context around the crash (a few lines above and below) for each frame, similar to single line bad-line is showing now

**Comment by davidmarkclements** [comment]
yah that sounds good

**Comment by davidmarkclements** [comment]
I'm also thinking you could have a web app, and a cute-stack plugin could feed every stack trace to this app
- useful for multi-process deployments
